atomic_reactor/plugins/post_tag_and_push.py

```diff
@@ -73,11 +73,12 @@ def run(self):
 
                 digests = get_manifest_digests(registry_image, registry,
                                                insecure, docker_push_secret)
-
                 tag = registry_image.to_str(registry=False)
                 push_conf_registry.digests[tag] = digests
 
+                push_conf_registry.config = get_config_from_registry(
```

Comment:
Adding a dynamic attribute like this will increase the difficulty of understanding the code base. I would've expected `DockerRegistry` to at least define it.